Source/WebCore/platform/gstreamer/GStreamerQuirkWesteros.cpp

```diff
@@ -51,17 +51,6 @@ GStreamerQuirkWesteros::GStreamerQuirkWesteros()
     }
 }
 
-void GStreamerQuirkWesteros::configureElement(GstElement* element, const OptionSet<ElementRuntimeCharacteristics>& characteristics)
-{
-    if (!characteristics.contains(ElementRuntimeCharacteristics::IsMediaStream))
-        return;
-
-    if (!g_strcmp0(G_OBJECT_TYPE_NAME(G_OBJECT(element)), "GstWesterosSink") && gstObjectHasProperty(element, "immediate-output")) {
-        GST_INFO("Enable 'immediate-output' in WesterosSink");
-        g_object_set(element, "immediate-output", TRUE, nullptr);
-    }
-}
-
```

Comment:
Hi @emutavchi! Digging a bit into the history behind this quirk, it seems to have been introduced by you [back in wpe-2.22](https://github.com/WebPlatformForEmbedded/WPEWebKit/pull/797/commits/f9377b1a7478c2b78d5fc64e65614f8ed4edf94d#diff-4376b854280205494dc9de0f08f985afd4d6584a7487c0a136adac2954c003cdR3863). Is the `immediate-output` property still functional on `westerossink`?

Context: testing Amazon Luna with this PR's branch (which is on top of wpe-2.38) on Sagemcom STB, the video stream freezes on the first frame when this property is set. Removing this quirk the video stream worked.

I also reproduced the freeze with `gst-launch-1.0 filesrc location=short.h264 ! h264parse ! brcmvidfilter ! westerossink immediate-output=TRUE`, which is fixed by setting `immediate-output=FALSE`

Test stream: [short.zip](https://github.com/user-attachments/files/16816147/short.zip)